frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -1580,6 +1593,13 @@ impl<T: Config> Pallet<T> {
 		Self::kill_snapshot();
 	}
 
+	// Returns `true` if the minimum number of blocks have passed since last successful election.
+	fn minimum_blocks_since_last_election() -> bool {
+		(frame_system::Pallet::<T>::block_number() - <LastElection<T>>::get()) >=
```

Comment:
Probably `Off` phase is not taken into account.

Reply:
Fixed in the last commit.